rest-api/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java

```diff
@@ -63,6 +64,11 @@
     Object execute(final OperationChain opChain);
 
     @POST
+    @Path("/chunked")
+    @ApiOperation(value = "Performs the given operation chain on the graph, returned chunked output", response = Element.class)
+    ChunkedOutput<String> executeChunked(final OperationChain<CloseableIterable<Element>> opChain);
```

Comment:
Please can you also add an example for this executeChunked method into 'SimpleExamplesService.java'